Document performance dashboard (#1300)
rancher/rancher-docs: docs/how-to-guides/advanced-user-guides/monitoring-alerting-guides/enable-monitoring.md

```

Then **Cert File Path** would be set to `/etc/alertmanager/secrets/cert.pem`.

## Rancher Performance Dashboard

> When monitoring is installed, you are given basic health metrics about the local Rancher server, such as CPU and memory data. To get advanced metrics for your local Rancher server, you must enable the Rancher Performance Dashboard for Grafana.

Reviewer: ```suggestion
When monitoring is installed on the upstream (local) cluster, you are given basic health metrics about the Rancher pods, such as CPU and memory data. To get advanced metrics for your local Rancher server, you must additionally enable the Rancher Performance Dashboard for Grafana.
```

I am not sure what the convention is about the local cluster - do we always call it local (like in the UI) or "upstream" (like in many places throughout the docs) or both?

Also clarified a bit what is available, as "server" is a quite generic word. I hope this helps

Author: Our existing usage is inconsistent, which is something we're trying to address with a [Glossary](https://github.com/rancher/rancher-docs/pull/1125) that's in the works. For this particular case, "local" is what we'll going forward to match what users see in the UI. Other existing variations will be updated eventually.

This dashboard provides access to the following advanced metrics:

- Handler Average Execution Times Over Last 5 Minutes
- Rancher API Average Request Times Over Last 5 Minutes
- Subscribe Average Request Times Over Last 5 Minutes
- Lasso Controller Work Queue Depth
- Number of Rancher Requests
- Number of Failed Rancher API Requests
- K8s Proxy Store Average Request Times Over Last 5 Minutes
- K8s Proxy Client Average Request Times Over Last 5 Minutes
- Cached Objects by GroupVersionKind
- Lasso Handler Executions
- Handler Executions over last 2 minutes
- Total Handler Executions with Error
- Data Transmitted by Remote Dialer Sessions
- Errors for Remote Dialer Sessions
- Remote Dialer Connections Removed

> - Remote Dialer Connections Added by Client

Reviewer 2: Are all of these, capitalization included, directly copied from the UI? Some of the capitalization is inconsistent (line 97 has "over" in lowercase while other metrics use "Over")

Author: Double checked dashboard names and updated. That one was a typo.

> - Remote Dialer Connections Removed (Top 20)
> - Remote Dialer Connections Added by Client (Top 20)

Reviewer: ```suggestion
- Remote Dialer Connections Active Connections (Top 20)
- Remote Dialer Connections Removed Connections Rate (Top 20)
- Remote Dialer Connections Added Connections Rate (Top 20)
```

Takes changes in https://github.com/rancher/charts/pull/3762 into account

Author: Since this is for 2.9, I'll address this one outside of this PR.

:::note

> Advanced heap analysis is not present as it is a very context-dependent technique that's meant for debugging and not intended for normal observation.

Reviewer: ```suggestion
Profiling data (such as advanced memory or CPU analysis) is not present as it is a very context-dependent technique that's meant for debugging and not intended for normal observation.
```

"Profiling" is the correct jargon here - I prefer being specific not to confuse expert readers. Still give an easier to understand definition for non-programmers.
